tests/sdk/v2/schema/test_compat.py

```diff
@@ -0,0 +1,176 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+"""
+Tests for ``orquestra.sdk.schema._compat``.
+"""
+import typing as t
+from pathlib import Path
+from unittest.mock import create_autospec
+
+import pytest
+
+from orquestra.sdk.schema import _compat, ir
+
+from .data import unpacking
+
+
+def _get_task_by_name(
+    wf_def: ir.WorkflowDef, task_name: str
+) -> t.Sequence[t.Tuple[ir.TaskDef, ir.TaskInvocation]]:
+    task_defs = [
+        task_def
+        for task_def in wf_def.tasks.values()
+        if task_def.fn_ref.function_name == task_name
+    ]
+    task_def_ids = {task_def.id for task_def in task_defs}
+
+    pairs = []
+    for task_def in task_defs:
+        invs = [
+            inv
+            for inv in wf_def.task_invocations.values()
+            if inv.task_id in task_def_ids
+        ]
+        pairs.extend([(task_def, inv) for inv in invs])
+    return pairs
+
+
+DATA_PATH = Path(__file__).parent / "data"
+SNAPSHOT_VERSIONS = [
+    # Last version without ``ir.WorkflowDef.metadata``
+    "0.44.0",
+    # Last version without ``ir.TaskDef.output_metadata``
+    "0.45.1",
+]
+
+
+class TestNOutputs:
+    class TestModernIR:
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏What is "modern IR"? "Current IR" seems to be more accurate

Reply:
Fixed in 5b9a0abce3